Examples/create-app.js

```diff
@@ -35,6 +35,15 @@ if (fs.existsSync(appName)) {
 let appNameAndroid = `${appName}-android`;
 let appNameIOS = `${appName}-ios`;
 let reactNativeVersion = args[1] || `react-native@${execSync('npm view react-native version')}`.trim();
+
+let reactNativeVersionBellowV049;
+if (reactNativeVersion) {
+    let reactNativeVersionNumberString = reactNativeVersion.split("@")[1];
```

Comment:
Lets move these two lines below that identifies if rn version is lower than certain rn version to separate function like `function isReactNativeVesionLowerThan(reactNativeCodePushVersion, versionToCheckFor)`

Reply:
Yes, it makes sense, I'll do this.